app/components/templates/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import Nav from '../../organisms/Nav/Nav';
+import Link from '../../../components/atoms/Link/Link.js'
+
+export default class Main extends React.Component {
+  constructor(props) {
+      super(props);
+  }
+
+  render() {
+    return(
+      <div className="app">
+        <Nav />
+        
+        <main>
```

Comment:
I think it's ok if we just put the component to keep it simple, I'd delete the `<main>` tag, as we are not using that kind of tags in the project, just to follow some rules and consistency. It's something we can add in the future . Thoughts? @Oliphaunte